tools/accuracy_checker/sample/sample_blob_config.yml

```diff
@@ -9,8 +9,6 @@ models:
         # Note: not all devices support blob execution, it is the reason why for sample we will use myriad.
         device: MYRIAD
```

Comment:
maybe we can also change the example device? Currently, CPU supports compiled blob format as well

Reply:
Thank you, sure!